src/python/uart.py

```diff
@@ -1,12 +1,18 @@
-import serial, requests, os
+import serial, os, socket, json
```

Comment:
Since this will likely need internet shouldn't we add a connectivity check here?

Reply:
The idea is this would store telemetry locally and optionally push it out to the web. If for some reason the internet went down or couldn't be accessed they could still access the data manually from the PI.

So I wouldn't want to do a internet check. Although now I'm wondering if the domain resolution would fail if the internet was down. If so I might want to check the internet and not try to send telemetry externally if it's down

Reply:
Added a `try`/`except` to the external service so it gets ignored during network failures